tools/kagami/src/main.rs

```diff
@@ -849,13 +853,69 @@ mod validator {
         }
     }
 
+    pub fn clone_default_validator(out_dir: impl AsRef<Path>) -> Outcome {
+        const GIT_ORIGIN: &str = "https://github.com/hyperledger/iroha.git";
+
+        // Sparse directory copy as suggested in https://stackoverflow.com/a/52269934/10669935
+        cmd!(
+            "git",
+            "clone",
+            "-n",
+            "--depth=1",
+            "--filter=tree:0",
+            "--branch=iroha2-dev",
```

Comment:
It might not necessarily be `iroha2-dev`.

Reply:
Remade it to employ swarm's functions.